docs/glossary.md

```diff
@@ -0,0 +1,175 @@
+ Glossary of Technical Terms
+---
+
+## A
+- [Account Aliases](#account-aliases)
+- [Admin Transaction](#admin-transaction)
+- [Auto Associations](#asynchronous)
+- [Auto-Renew Accounts](#auto-renew-accounts)
+- [Address Book](#address-book)
+- [Application Accounts](#application-accounts)
+
+## B
+- [BLS](#bls)
+- [Block Proof](#block-proof)
+- [Block Signature](#block-signature)
+- [Block Stream](#block-stream)
+
+## C
+- [Civilian Accounts](#civilian-accounts)
+- [Config.txt](#config.txt)
+- [Congestion Pricing](#congestion-pricing)
+- [Consensus Time](#consensus-time)
+
+## D
+- [Delegate Call](#delegate-call)
+
+## E
+- [EOA](#eoa)
+
+## H
+- [Hollow Accounts](#hollow-accounts)
+
+## L
+- [Leaky Tests](#leaky-tests)
+
+## P
+- [Precompiles](#precompiles)
+
+## S
+- [Security Model V2](#security-model-v2)
+- [Sidecars](#sidecars)
+- [Synthetic Transaction](#synthetic-transactions)
+- [System Contracts](#system-contracts)
+- [System Transaction](#system-transaction)
+
+## T
+- [Token Allowance](#token-allowance)
+
+## W
+- [Weibar](#weibar)
+
+---
+
+## Address Book
+**Address Book**: The _**network**_ address book is data structure that contains information (such as node ID, alias, IP address, and weight) for all nodes in the network, 
+not just consensus, but also block nodes, future validator nodes and more.
+The address book is stored as a file on disk at the moment but will be stored purely in state as part of Dynamic Address Book phase 3.	Disambiguation: There is also an "address book" in files 0.0.101 and 0.0.102 which is different to the "network address book" which are manually managed by devops.
```

Comment:
Just a note, your spacing is still off here (there's a tab in place of a space after the first sentence).
Tip: setting your tab width in github to 16 makes tabs super easy to spot.

Reply:
Thanks @jsync-swirlds. I have opened a separate issue https://github.com/hashgraph/hedera-services/issues/14016 to address